packages/hash/frontend/src/components/hooks/useReorderPage.ts

```diff
@@ -16,15 +20,19 @@ export const useReorderPage = () => {
     refetchQueries: ({ data }) => [
       {
         query: getAccountPagesTree,
-        variables: { ownedById: data!.setParentPersistedPage.ownedById },
+        variables: {
+          ownedById: extractOwnedByIdFromEntityId(
+            data!.setParentPersistedPage.metadata.editionId.baseId,
```

Comment:
Do we want to do the same as before for `useCreatePage` to check for empty data?